scripts/generate-and-apply-contour-yaml.sh

```diff
@@ -30,4 +30,12 @@ kubectl apply -f tkg-extensions/ingress/contour/aws/02-service-contour.yaml
 kubectl apply -f tkg-extensions/ingress/contour/aws/02-service-envoy.yaml
 kubectl apply -f tkg-extensions/ingress/contour/aws/03-contour.yaml
 kubectl apply -f tkg-extensions/ingress/contour/aws/03-envoy.yaml
-sleep 10s #Wait a sec to get DNS/IP assigned
\ No newline at end of file
+
+# Wait until DNS/IP assigned
+echo -n "Waiting for Envoy IP Assignment"
+while [ "" = "$(kubectl get svc envoy -n tanzu-system-ingress -o jsonpath='{.status.loadBalancer.ingress[0].ip}')" ]; do
```

Comment:
This won't work on AWS, since the ELB info is in `.status.loadBalancer.ingress[0].hostname`
We need to check the iaas, as we do here https://github.com/Pivotal-Field-Engineering/tkg-lab/blob/master/scripts/update-dns-records-route53.sh#L12-L21